state.gd

```diff
@@ -0,0 +1,45 @@
+extends Node
+
+var players = []
+var queue = []
+var start
```

Comment:
The start field does not contain any state. I am not even sure it is used anywhere but I might missed it.

Reply:
Will be needed as a teleport target for the "go to start action" not implemented yet